two_factor/utils.py

```diff
@@ -58,3 +59,6 @@ def totp_digits():
     for totp tokens. Defaults to 6
     """
     return getattr(settings, 'TWO_FACTOR_TOTP_DIGITS', 6)
+
+
+def should_display_dropdown_menu(request): return import_string("django_admin_index.utils.should_display_dropdown_menu") and request.user.is_verified()
```

Comment:
please write return in the next line. 
And it looks like `should_display_dropdown_menu` function is imported but not called

Reply:
def should_display_dropdown_menu(request):
    return import_string("django_admin_index.utils.should_display_dropdown_menu")(request) and request.user.is_verified()

ADMIN_DISPLAY_DROP_DOWN_MENU_CONDITION_FUNCTION = _path to the function_
this peace of code will be needed in open forms then

Reply:
I think in this particular case the dropdown would not be shown since the login page has unique styling. But yes, if openforms needs to hide drop-down somewhere it should be implemented in that project

Reply:
Correct - it's up to the end-project using the libraries to do the final configuration. We can easily set this up by default in our default-project (and should do so).